README.md

```
@@ -105,6 +105,55 @@ for dt in Quarter(date(1958, 3, 25)):
+The `Week` class itself is also iterable, for example:
+
+```
+for week in Week:
+    print(week)
+```
+
+will start enumerating over all weeks since 0001-01-01.
+
+A time unit also has a length: the number of time units that can be represented, so:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)